pkg/runner/runner.go

```diff
@@ -234,15 +234,21 @@ func (r *Runner) Run(ctx context.Context) error {
 		}
 
 		// Remove the PID file if it exists
-		if err := process.RemovePIDFile(r.Config.BaseName); err != nil {
-			logger.Warnf("Warning: Failed to remove PID file: %v", err)
+		// Skip PID files in Kubernetes - pods are managed by the cluster
```

Comment:
Can you leave these checks out? I want to refactor the handling of PID files behind some interfaces to avoid needing to check for k8s every time we need to do something with pid files.